deepchecks/utils/single_sample_metrics.py

```diff
@@ -34,7 +34,7 @@ def calculate_neg_cross_entropy_per_sample(labels, probas: np.ndarray, model_cla
     if model_classes is not None:
         if any(x not in model_classes for x in labels):
             raise DeepchecksValueError(
-                f'Label observed values {sorted(labels.unique())} contain values '
+                f'Label observed values {sorted(np.unique(labels))} contain values '
```

Comment:
make sure print is of a list and not a numpy array

Reply:
its a list